libr/core/disasm.c

```diff
@@ -1613,7 +1619,7 @@ static void ds_pre_xrefs(RDisasmState *ds, bool no_fcnlines) {
 	ds->line_col = ds->prev_line_col;
 	ds_print_lines_left (ds);
 	if (!ds->show_offset && ds->show_marks) {
-		ds_print_show_cursor (ds);
+		ds_print_show_cursor (ds, 0);
```

Comment:
```suggestion
		ds_print_show_cursor (ds, false);
```

Reply:
See ^ please

Reply:
I fixed this already but the commit got lost becausr when gh updates a change it breaks the whole commit history and i purge it and chcerrypick one by one when pulling

Reply:
Please re-commit it. GH does not break the history, most likely you rebased before pulling and getting the newer commit commited directly by GH.

Reply:
done